### ISSUE TITLE
Add GNOME 45 Support

### ISSUE THREAD

**Issue by rubin55** [BjoernDaase/noannoyance#31]
As title!

**Comment by ivulfson** [comment]
Pretty please with a cherry on top!

**Comment by lens0021** [comment]
I've tried the forked version and it works https://extensions.gnome.org/extension/6109/noannoyance-fork/

**Comment by rubin55** [comment]
> I've tried the forked version and it works https://extensions.gnome.org/extension/6109/noannoyance-fork/

Yes, but it has to be noted that every time the topbar hides, your `journalctl` log is spammed with Javascript error(s).
